_includes/checkpoints/checkpoint2.py

```diff
@@ -1,19 +1,38 @@
-# need to import the "math" module to use different math functions
+"""
+Checkpoint 2: Modularize a repeated problem
+-------------------------------------------
+
+Create a function to calculate the result for any initial flux, shield
+thickness, and attenuation coefficient.
+"""
+
 import math as m
 
+
 def calc_phi(phi_0, mu, x):
+    """Calculates final flux after shield of thickness x with
```

Comment:
Wondering if docstrings for functions should be a separate checkpoint or not. Whatcha think?

Reply:
I don't think this necessarily needs to be a separate script but we could introduce it in two stages. Have them write a function and have them document it. I am hesitant to make it a separate checkpoint because I don't think the two should be separate in practice at all. Also, when introducing functions in the first half they will be introduced to documentation (I think..).

Reply:
Sounds good.